src/main/java/org/sagebionetworks/web/client/presenter/LoginPresenter.java

```diff
@@ -137,8 +133,6 @@ public void showView(final LoginPlace place) {
 			showTermsOfUse(false);
 		} else if (SHOW_SIGNED_TOU.equals(token) && authenticationController.getCurrentUserAccessToken() != null) {
 			showTermsOfUse(true);
-		} else if (!recognizedTokens.contains(token) && !"".equals(token) && token != null) {

```

Comment:
Previously assumed that if we did not recognize the token then we should attempt to use it as an access token.  Removed